### ISSUE TITLE
[meeting] Timetable oemof spring user/dev meeting

### ISSUE THREAD

**Issue by joroeder** [oemof/oemof#73]
This issue discusses the time table. I think it would be nice, if we can fix the sessions, which are announced in the wiki, until Wednesday next week (6th of May), and then, send a newsletter with the link to the agenda/wiki (So, the results of this issue will be transferred to the wiki, and the timeslots of these sessions should not be changed anymore.). I assume that the people who are really interested in the oemof meeting have kept their calendar free these days (13-15th of May). So, this should result in a lean discussion, where especially the chairs' feedback is wanted, because they must have time in their own session 😊 Of course, general feedback is welcome as well 😉

The idea is, that there will be an (extra) etherpad, where everybody can then from Wednesday on write his/her name to each session he/she wants to participate. So, the chair of each session have already an idea about the number of participants, which might influence the format or some technical issues of the session.

So, here comes the suggestion of the agenda:
[https://ethercalc.org/b2zg1e72npzv](https://ethercalc.org/b2zg1e72npzv)

Some notes:
* I made a standard slot time of 45 min. two slots in the morning, three in the afternoon, which a nice 2h lunch break in between
* to relax it further, there is a 30 min break between each session in the afternoon, so it will be no problem, if a session needs some 15 min extra, and it is nice to have a break. In the mornging, a 15 min break should be fine.
* there is a welcome and a closure session
* at the moment there is no session parallel (that does not need to stay like that). That is the benefit for all who kept the deadline for annoucing sessions 😉
* there are columns U: user-focus D: dev-focus U&D: interesting for everybody. Of course, everybody can join every session. this is more as hint how is adressed by each session.
* on Thursday, there is an extra option of a second parallel dev session
* at the moment, there are no slots for recap the sessions. I think we don't need them in this virtual meeting, they simply lead to more time in webkonferences. Since there are no parallel sessions at the moment, everybody can join all sessions he wants anyway. Or maybe we just do a "mid-session" corresponding the welcome and closure on Thursday lunch time?!
* altogether, I think it is not too dense for now, which is fine. either some people like to do some spontaneous small sessions in between, or it is just more relaxed.

Could all chairs please give a short feedback? (@jnnr @jokochems @c-moeller @Ludee @nesnoj @busiing @birgits @uvchik ) Thanks!

**Comment by jokochems** [comment]
Hi, thanks a lot for the timetable draft.
The slots proposed work out fine for me.

**Comment by nesnoj** [comment]
Looks good to me, thank you @joroeder !

**Comment by birgits** [comment]
Works for me! Thank you @joroeder

**Comment by c-moeller** [comment]
Works for me, too. Thank you!

**Comment by busiing** [comment]
@joroeder thank you for your work!
I am a little concerned with my slot, bc the last meeting showed that the oemof community likes to discuss. Additionally, the slot is names website.
I can present the contributions of @beierd, @c-moeller and me.

But the biggy is the foundation session. We are lacking commitment for some tasks, feedback and input for decisions and atm even lack discussion. I think this will change as soon as we start the session like last time. 
tl;dr: Let's skip the website session (maybe show the changes within 5 minutes at some point). And I think we might need more time for the foundation topic. It might be convenient to make a follow-up hands-on session if enough people are motivated enough.

**Comment by uvchik** [comment]
Please edit the Program here:

https://ethercalc.org/b2zg1e72npzv

**Comment by joroeder** [comment]
Thank you all for your Feedback! And thank to all chairs for your session! @uvchik thanks for the nice appearance now 😉

@jnnr @Ludee is the timeslot of your session fine for you?

@busiing Sorry, since you named your session "website" I thought one timeslot is fine. But sure, discussing the foundation demands for some more time, or is that 1:15 h slot fine? Otherwise, we could add an evening timeslot for that on Thursday (which could go into something informal with beer as well)? All other slots are filled, and I don't want to put that somewhere parallel. Any other ideas?

When now all chairs are fine with their slot, I would suggest to send a newsletter with the link the table. Who can do that? I don't have access to the wordpress website stuff ...
Then, the existing sessions should not be shifted anymore, and the people are encouraged to put their names to the session, they want to join. Therefore, I have added the column "participants" to the sessions table. Do you think that approach is fine? any objections to proceed like that?

**Comment by uvchik** [comment]
@c-moeller, @birgits, @jokochems, @nesnoj, @Ludee, @jnnr, @busiing 

Please check if you are fine with color, description, name,... and change it if necessary. 

> table. Who can do that? I don't have access to the wordpress website stuff ...

@beierd said, that he will do that.

**Comment by fwitte** [comment]
This may have been discussed before, but I am not sure at the moment: The oemof_developer has admin rights to all repositories of the organisation and I just recieved an e-mail that the password was reset. I am sure, it was someone of the devs. But as so many people have access to that account, does that not pose a potential security issue? Should we do something about that and is there a way to prevent this without loosing the advantages of having such an account?

Also, what about mandatory 2FA for all people with admin rights to the organisation or single repositories? I am not using it either, at the moment.

These topics could be added to the agenda for a short discussion/decision finding, if desired.

Have a nice day

**Comment by beierd** [comment]
It should be published now

**Comment by uvchik** [comment]
I think we need time for decisions that come up during the days. Furthermore we need a PR auction. We do have 23 open PR and at least half of them are not active any more.

We may meet 30 minutes before the official beginning and talk about some organisational stuff.

**Comment by smartie2076** [comment]
Hi @uvchik, the link to the [timetable](https://ethercalc.org/b2zg1e72npzv) does not work anymore (505 Bad Gateway). Do you have a backup and can share it again?

Also, can you limit the the Project Pitches to meeting room "Session 1" so that that is clear?

**Comment by smartie2076** [comment]
> 
> 
> Hi @uvchik, the link to the [timetable](https://ethercalc.org/b2zg1e72npzv) does not work anymore (505 Bad Gateway). Do you have a backup and can share it again?
> 
> Also, can you limit the the Project Pitches to meeting room "Session 1" so that that is clear?

The timetable is back now and I adapted it for the Project Pitches.

**Comment by busiing** [comment]
Closed - until necessary again.
